README.md

```diff
@@ -28,6 +28,9 @@ metadata:
 
 ## Running the controller in a cluster
 
+The controller requires [cert-manager](https://cert-manager.io) running in the cluster. You can install it using `make install_cert_manager` if you 
```

Comment:
That's needed for Kubernetes cluster only. On Openshift we use openshift out-of-the-box mechanism to get the certificates for the webhook server.